web/public/mgmt/config.yml

```diff
@@ -3090,16 +3090,22 @@ collections:
           options: ["expiration", "renewal"],
         }
       - { label: "License Name", name: "licenseName", widget: hidden, default: "" }
-      - { label: "Renewal Event Label", name: "renewalEventDisplayName", widget: "string", required: true }
       - { label: "URL Slug", name: "urlSlug", widget: "write-once-read-only-no-space", required: true }
       - {
           label: "Expiration Event Label",
           name: "expirationEventDisplayName",
           widget: "string",
           required: true,
         }
+      - { label: "Renewal Event Label", name: "renewalEventDisplayName", widget: "string", required: true }
+      - {
+          label: "Above License Status Content",
+          name: "aboveLicenseStatusContent",
+          widget: "markdown",
+          required: true,
```

Comment:
Should this be `false`? Will every instance have content here?

Reply:
Yes, based on the designs, there is a license status card that would go in between the content. Based on the discussions we have had so far, it is my understanding they will always be content above and below the card.

Reply:
https://www.figma.com/design/Xr5I96TLqdoE6ZAorDTUUi/56-60-Business-Experience-Sprint-Designs?node-id=40784-46405&t=PEBE0BFJICY0t4m3-1